tests/lib/rules/jsx-newline.js

```diff
@@ -166,6 +166,21 @@ const tests = {
         {messageId: 'newLine'},
         {messageId: 'newLine'}
       ]
+    },
+    {
+      code: `
+      <div>
+
+       <button></button>
+       
+      </div>
+      `,
+      options: [{
+        removeNewLines: true
+      }],
+      errors: [{
+        messageId: 'noNewLine'
+      }]
```

Comment:
```
  1) jsx-newline
       invalid
         
      <div>

       <button></button>
       
      </div>
      :

      AssertionError [ERR_ASSERTION]: Should have 1 error but had 0: []
      + expected - actual

      -0
      +1
      
      at testInvalidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:666:24)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:893:25)
      at processImmediate (internal/timers.js:439:21)
```

Reply:
Even when the error is being reported it is saying not reported. I am a bit confused here too :/

Reply:
it's an "invalid" test but it has 0 errors, but it should have 1.